lib/pull_request.rb

```diff
@@ -125,14 +129,32 @@ def tell_dependency_manager_what_dependencies_are_allowed
   end
 
   def tell_dependency_manager_what_dependabot_is_changing
+    dependency_updates = commit_message.scan(/Updates `(\w+)` from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d)/)
+
+    # Commit messages can have different formats
+    if dependency_updates.empty?
+      dependency_updates = commit_message.scan(/Bump (?:\[.*?\]\(.+?\) )?(\w+) from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d)/)
+    end
+
+    mentioned_dependencies = {}
+
+    dependency_updates.each do |name, from_version, to_version|
+      mentioned_dependencies[name] = [from_version, to_version]
```

Comment:
I think it would be cleaner to assign this to a hash:

```suggestion
      mentioned_dependencies[name] = {
        from_version:,
        to_version:,
      }
```